buildsystem/modules/FindPython.cmake

```diff
@@ -43,6 +43,29 @@ function(py_get_config_var VAR RESULTVAR)
 	set("${RESULTVAR}" "${RESULT}" PARENT_SCOPE)
 endfunction()
 
+function(py_get_lib_name RESULTVAR)
+	# uses py_exec to compute Python's C/C++ library name, just like python-config does.
+	py_get_config_var(VERSION PYTHON_VERSION)
+	if(NOT "${PYTHON_VERSION}" VERSION_LESS "3.2")
+		set(USE_ABIFLAGS "use")
```

Comment:
True that we don't support it. However, if we invoke this on an older version of python (which will likely be called, since we are searching through all interpreters) then python fails because it can't find sys.abiflags. This behavior changed in version 3.2